js/ui/appIconBar.js

```diff
@@ -466,27 +472,27 @@ const AppIconButton = new Lang.Class({
 
         // Calculate location of the label only if we're not tweening as the
         // values will be inaccurate
-        if (!Tweener.isTweening(this.actor)) {
-            let iconMidpoint = this.actor.get_transformed_position()[0] + this.actor.width / 2;
+        if (!Tweener.isTweening(this)) {
+            let iconMidpoint = this.get_transformed_position()[0] + this.width / 2;
```

Comment:
Would be nice to save the output of `this.get_transformed_position()` into local variables, since it's used below too.